### ISSUE TITLE
Implement units for functions

### ISSUE THREAD

**Issue by Jeppembo** [vitaproject/core#22]
There have been several bugs with mixing units, especially mm in Eich functions and m for device size, which gives wrong outputs for power profiles and total power output.

**Comment by daniel-iglesias** [comment]
True, I think it'd be better changing everything to meters, even if we get far too many zeros sometimes. What do you think?

**Comment by Jeppembo** [comment]
That's what I've done so far. But I am considering implementing astropy.units to make more robust checks and give an error if one tries to mix units. This will, however, require a lot of adaptations for the current scripts and I'm not sure it's worth the time required for it yet.

**Comment by mattngc** [comment]
I strongly support this move. It is far better to standardise on SI units and use them throughout. You can always do the unit conversions in utility functions when displaying values nicely to users. We do this in a few places in Cherab.

The astropy idea is interesting, but it's a heavy dependency for a small amount of functionality. I would assess how much you need it before going down that path. Just standardising on SI across the package may be enough.

**Comment by daniel-iglesias** [comment]
Let's change all variables to SI then. Actually moving from MW to W will be worse than from mm to m :smile:
